docs/online/src/scenarios/postmessage.rst

```diff
@@ -269,6 +270,76 @@ You can send the following messages; all others are ignored:
         }
 
 
+..  data:: Host_RefreshSessionInfo
+
+    When the host wants to send new tokens to |wac|, for instance when an :term:`access token` is about to expire, 
+    a :data:`Host_RefreshSessionInfo` message is posted. The host can send a set of new access tokens and their properties 
+    in this post message. 
+
+    ..  note:: 
+        |wac| expects a :data:`Host_RefreshSessionInfo` response to a :data:`App_RefreshSessionInfo` request. If no response is 
+        received by |wac|, the session could expire. 
+    
+    If the host is unable to issue new tokens an error message must be provided, with an optional description of the 
```

Comment:
How is the error used? If it is a string, how are you ensuring that the host does not return you private info, like the name of the document in the error?